frontend/src/components/common/checkboxes/CheckBox.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "./CheckBox.css";
+
+export interface CheckBoxStyleParams {
+  size?: string;
+  selectedColor?: string;
+  unselectedColor?: string;
+  disabled?: boolean;
+}
+
+interface CheckBoxParams extends CheckBoxStyleParams {
+  onClick: () => void;
+  isSelected: boolean;
+  radio?: boolean;
+}
+
+const CheckBox = ({
+  onClick,
+  isSelected = false,
+  radio = false,
+  size = "22px",
+  selectedColor = "#94D969",
+  unselectedColor = "#ddd",
+  disabled = false,
+}: CheckBoxParams): React.ReactElement => {
+  return (
+    <div className="container" onClick={disabled ? undefined : onClick}>
```

Comment:
Can you double check if there are aria- attributes we should add for a11y? There may not be but I just wanted to make sure we're thinking about it.